demisto_sdk/commands/common/hook_validations/base_validator.py

```diff
@@ -130,3 +132,128 @@ def _is_id_equals_name(self, file_type):
             print_error(Errors.suggest_fix(self.file_path))
             return False
         return True
+
+    def _load_conf_file(self):
+        with open(self.CONF_PATH) as data_file:
+            return json.load(data_file)
+
+    def are_tests_configured(self) -> bool:
+        """
+        Checks if a file (playbook or integration) has a TestPlaybook and if the TestPlaybook is configured in conf.json
+        And prints an error message accordingly
+        """
+        file_type = self.structure_validator.scheme_name
+        tests = self.current_file.get('tests', [])
+        if not self.yml_has_test_key(tests, file_type):
```

Comment:
This treats the situation where there is no `tests` key in the YAML and the situation in which there is a `tests` key in the YAML file whose value is empty the same way. Is that intentional?

Reply:
Yes
We should either add **No tests** or add actual tests
That's the convention as I see it